services/websocket/service.go

```diff
@@ -0,0 +1,200 @@
+package websocket
+
+/*
+ */
+
+import (
+	"net/http"
+
+	"strconv"
+
+	"fmt"
+
+	"encoding/binary"
+
+	"github.com/dedis/cothority/log"
+	"github.com/dedis/cothority/network"
+	"github.com/dedis/cothority/sda"
+	"github.com/dedis/cothority/services/status"
+	"golang.org/x/net/websocket"
```

Comment:
Ok I believe this file also must be removed, still using the old websocket thing.